servicemesh-install/topics/install-prerequisites.adoc

```diff
@@ -236,22 +267,6 @@ $ oc ex config patch master-config.yaml.prepatch -p "$(cat master-config.patch)"
 $ master-restart api
 $ master-restart controllers
 ```
-+
-. Then you must modify each individual deployment that you want to monitor as part of your service mesh to enable automatic sidecar injection. Each deployment where you want to enable automatic injection needs to contain the `sidecar.istio.io/inject: "true":` annotation.
```

Comment:
I don't necessarily think this belongs here, but this is the only place on the OSSM docs that mentions the annotation. I don't think we can entirely remove this.

Reply:
This is more to do with how to deploy applications and take advantage of the automatic injection so has nothing to do with installation.  It definitely shouldn't be in this section, perhaps we need to introduce a new one covering how to develop applications using automatic injection?

Reply:
I agree. A separate section would be great. Just want to make sure this makes it into the doc.